success.md

```diff
@@ -3,4 +3,13 @@ layout: default
 title: Success
 ---
 
-# Success
+# Let's file your complaint.
```

Comment:
This looks good for now. I may poke around for some kind of graphic here to reinforce the messaging. Or maybe an icon at least.